src/database/parse/parse-event-database.ts

```diff
@@ -0,0 +1,61 @@
+import EventDatabase from "../event/event-database";
+import EventModel from "../event/event-model";
+
+/**
+ * `ParseEventDatabase` is an `EventDatabase` implementation that uses the Parse JavaScript SDK.
+ * This class should be instantiated using `ParseDatabaseFactory`.
+ */
+export default class ParseEventDatabase implements EventDatabase {
```

Comment:
This class is essentially the same as the previous `ParseServer` class, except the Parse configuration logic now resides in the `ParseDatabaseFactory` class.